docker/dev/Dockerfile

```diff
@@ -35,7 +36,7 @@ RUN tar -xf /opt/thrift-0.13.0.tar.gz -C /opt; \
     --without-dotnetcore --without-d --without-qt4 --without-qt5;    \
     cd /; \
     make -C /opt/thrift-0.13.0 -j $(nproc) --silent install; \
-    rm -rf /opt/thrift-0.13.0 /opt/thrift-0.13.0.tar.gz
+    rm -rf /opt/thrift-0.13.0
```

Comment:
Where did the targz go?

Reply:
It was pointed out by @csordasmarton that deleting a file in a separate layer does not decrease the container size, so it was unnecessary.

Reply:
Ah, right. Either way, to reduce the size of the container, you could add the curl command fetching the file to this `RUN`, or use the "multi-stage build" thing where you have multiple `FROM`s and in one container build Thrift and in the other one just copy the installation over.